src/test/java/com/ksm/wstbackoffice/controller/CountryControllerIT.java

```diff
@@ -37,6 +40,19 @@
         );
     }
 
+    public static Stream<Arguments> findCountriesStartWithTestArguments() {
+        return Stream.of(
+                Arguments.of(Arrays.asList(), 400),
+                Arguments.of(Arrays.asList(""), 200),
+                Arguments.of(Arrays.asList(" "), 200),
+                Arguments.of(Arrays.asList("a"), 200),
+                Arguments.of(Arrays.asList("a "), 200),
+                Arguments.of(Arrays.asList("a","a"), 200),
+                Arguments.of(Arrays.asList("a", "ua", "b"), 200),
+                Arguments.of(Arrays.asList("a","au"), 200)
```

Comment:
This testing is not enough. Although it tests the correct HTTP response status code, it does not test the result.
I'd like you to think about how we can test the countries that our API returns.
E.g. for "sa" I am expecting to receive this
Saint Kitts and Nevis
Saint Lucia
Saint Vincent and the Grenadines
Samoa
San Marino
Sao Tome and Principe
Saudi Arabia

Reply:
done